packages/core/admin/admin/src/content-manager/pages/ListView/FieldPicker/utils/tests/getAllAllowedHeaders.test.js

```diff
@@ -1,21 +0,0 @@
-import getAllAllowedHeaders from '../getAllAllowedHeader';
-
-describe('CONTENT MANAGER | containers | ListView | utils | getAllAllowedHeaders', () => {
```

Comment:
Where did the test go?

Reply:
I moved the function into the `FieldPicker` the only place it's used.